libclamav_rust/src/css_image_extract.rs

```diff
@@ -264,7 +265,7 @@ impl<'a> Iterator for CssImageExtractor<'a> {
         // Find the next base64 encoded image
         if let Some(base64_image) = self.next_base64_image() {
             // Decode the base64 encoded image
-            match base64::decode(base64_image)
+            match general_purpose::STANDARD.decode(base64_image)
                 .map_err(|e| CssExtractError::Base64Decode(format!("{}", e)))
```

Comment:
You seem to be throwing away this error, since matching on it just evaluates to `None`

If this is what you intended, this is a simpler construct (no `match`):
``` rust
general_purpose::STANDARD.decode(base64_image).ok()
```
That discards the error, converting the result to `Some(image)` upon success, or `None` upon error.

Reply:
Oh I like this. Thank you!